<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particulas</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Agregar</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="14" column="0">
       <widget class="QLineEdit" name="velocidad"/>
      </item>
      <item row="23" column="0">
       <widget class="QPushButton" name="mostrar">
        <property name="text">
         <string>Mostrar</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0">
       <widget class="QSpinBox" name="color_r">
        <property name="maximum">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QSpinBox" name="color_b">
        <property name="maximum">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="id"/>
      </item>
      <item row="12" column="0">
       <widget class="QLineEdit" name="destinoy"/>
      </item>
      <item row="19" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Green</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QSpinBox" name="color_g">
        <property name="maximum">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="origenx"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Color RGB&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Velocidad&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLineEdit" name="destinox"/>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="origeny"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Origen&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Destino&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
      </item>
      <item row="22" column="0">
       <widget class="QPushButton" name="agregar">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="22">
       <widget class="QPlainTextEdit" name="salida"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 7pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Captura de Particulas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tabla</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="buscar_lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="buscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tabla"/>
      </item>
      <item>
       <widget class="QPushButton" name="mostar_tabla">
        <property name="text">
         <string>Mostrar Tabla</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Grafica</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>321</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>311</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Origen X</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="ordenarV">
         <property name="text">
          <string>Ordenar Velocidad</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="dibujar">
         <property name="text">
          <string>Dibujar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="mas_cercano">
         <property name="text">
          <string>Particulas m√°s Cercana</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="origenX"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Destino X</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLineEdit" name="destinoX"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="ordenarD">
         <property name="text">
          <string>Ordenar Distancia</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Destino Y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="limpiar">
         <property name="text">
          <string>Limpiar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="puntos">
         <property name="text">
          <string>Puntos</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="origenY"/>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Origen Y</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="destinoY"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuGrafos">
    <property name="title">
     <string>Grafos</string>
    </property>
    <addaction name="actionVer"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos">
    <property name="title">
     <string>Algoritmos</string>
    </property>
    <addaction name="actionProfundidad"/>
    <addaction name="actionAmplitud"/>
    <addaction name="actionPrim"/>
    <addaction name="actionKrustal"/>
    <addaction name="actionDijkstra"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuGrafos"/>
   <addaction name="menuAlgoritmos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionVer">
   <property name="text">
    <string>Ver</string>
   </property>
  </action>
  <action name="actionDibujar">
   <property name="text">
    <string>Dibujar</string>
   </property>
  </action>
  <action name="actionProfundidad">
   <property name="text">
    <string>Recorrido Profundidad</string>
   </property>
  </action>
  <action name="actionPrim">
   <property name="text">
    <string>Prim</string>
   </property>
  </action>
  <action name="actionKrustal">
   <property name="text">
    <string>Krustal</string>
   </property>
  </action>
  <action name="actionDijkstra">
   <property name="text">
    <string>Dijkstra</string>
   </property>
  </action>
  <action name="actionAmplitud">
   <property name="text">
    <string>Recorrido Amplitud</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
